packages/story-editor/src/components/canvas/canvasLayout.js

```diff
@@ -18,14 +18,20 @@
  * External dependencies
  */
 import styled, { StyleSheetManager } from 'styled-components';
-import { memo, useRef, useCallback } from '@web-stories-wp/react';
+import { memo, useRef } from '@web-stories-wp/react';
 import { __ } from '@web-stories-wp/i18n';
 import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
-import { useCanvas, useLayout } from '../../app';
+import { useComposeRefs } from '@web-stories-wp/design-system';
```

Comment:
Not related to this PR perhaps but wondering what's the difference between `useCombinedRefs` and `useComposeRefs`? First time seeing `useComposeRefs` in the `story-editor` package, so started wondering. Thoughts?

Reply:
they look like duplicates. Thanks for calling out. I'll create a little ticket to consolidate them

Reply:
Here's that ticket:

https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/10093